core/controllers/base.py

```diff
@@ -330,6 +330,8 @@ def render_template(
                 self.username and self.username in
                 config_domain.WHITELISTED_COLLECTION_EDITOR_USERNAMES.value
             ),
+            'promo_bar_enabled': config_domain.PROMO_BAR_ENABLED.value,
```

Comment:
OK to file an issue to remove this after migration?

The reason I ask is: fetching values from config_domain takes datastore cycles. If this happens every request, it is a hit on speed.